src/index.ts

```diff
@@ -323,10 +324,10 @@ export class GoogleToken {
       const r = await request<TokenData>({
         method: 'POST',
         url: GOOGLE_TOKEN_URL,
-        data: {
+        data: qs.stringify({
```

Comment:
@bcoe If the documentation on the endpoint is right - I think this would be the fix.  Gaxios sees `object` here, and just stomps the content-type header (which is bad), but this should stop it.